Tests/Time/TimeTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Framework\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Zest\Time\Time;
+
+class TimeTest extends TestCase
+{
+    public function testLeapYear()
+    {
+        $this->assertTrue(Time::isLeapYear(2020));
+        $this->assertFalse(Time::isLeapYear(2019));
+    }
+
+    public function testNow()
+    {
+        $this->assertIsInt(Time::now());
+        $this->assertIsInt(Time::now('Asia/Taipei'));
+    }
+
+    public function daysInMonthDataProvider()
+    {
+        return [
+            [2, 2019, 28],
+            [3, 2019, 31],
+            [4, 2019, 30],
+            [2, 2012, 29],
+            [2, 2016, 29],
+        ];
+    }
+
+    /**
+     * @dataProvider daysInMonthDataProvider
+     */
+    public function testDaysInMonth($month, $year, $expected)
+    {
+        $this->assertSame($expected, Time::daysInMonth($month, $year));
+    }
+
+    public function formatsSecondsDataProvider()
+    {
+        return [
+            [1554339600, '09:00:00'],
+            [1554343200, '10:00:00'],
+        ];
+    }
+
+    /**
+     * @dataProvider formatsSecondsDataProvider
+     */
+    public function testFormatsSecondsDataProvider($seconds, $expected)
+    {
+        $this->assertSame($expected, Time::formatsSeconds($seconds, 'Asia/Taipei'));
+    }
+}
```

Comment:
It needs more tests.

For example, the `ago`, `formatsSeconds` and so on.

Reply:
yes you can send PR

Reply:
i see but i think it will not work with timestamp

Reply:
@Lablnet, thank you for your suggestion.

I think we can create another issue about enhancing the `Time` class unit tests after this PR is merged.

This PR I think we're focus on implementing  the `Time` classes completely.

Reply:
Yes its good grate thanks